modules/apps/portal-language/portal-language-override-service/src/main/java/com/liferay/portal/language/override/service/impl/PLOEntryServiceImpl.java

```diff
@@ -86,6 +89,20 @@ public int getPLOEntriesCount(long companyId) throws PortalException {
 		return ploEntryLocalService.getPLOEntriesCount(companyId);
 	}
 
+	@Override
+	public void importPLOEntries(InputStream inputStream, String languageId)
+		throws IOException, PortalException {
+
+		PermissionChecker permissionChecker = getPermissionChecker();
+
+		PortalPermissionUtil.check(
+			permissionChecker, PLOActionKeys.MANAGE_LANGUAGE_OVERRIDES);
+
+		ploEntryLocalService.importPLOEntries(
+			permissionChecker.getCompanyId(), inputStream, languageId,
+			permissionChecker.getUserId());
```

Comment:
I'd consider adding an exception mapper for this if there is not one already. An ExceptionMapper translates a Java exception to an API Response error/problem. Otherwise, the user will just get back a generic "something went wrong 🤷‍♂️" message in their response.

Reply:
That makes sense. We don't have `*PLOEntryExceptionMapper` for each of the possible exceptions (Key, LanguageId, Value and now the File)

So I'll add them